ui/raidboss/data/05-shb/raid/e9s.txt

```diff
@@ -1,10 +1,78 @@
 ### Eden 9 (Savage)
 # Eden's Promise: Umbra (Savage)
+# -ii 5B19 5609 5627 560F 5619 5621 55FA 5A95 561E 5614 55FF 561B 5600 5B01 55FC 5606 55FF 55FD 5616 5615 5B00 5611 5610 510E 5B01 561C 5B07 560A 5624 5617 562B 5019 561D
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
 
-0 "Start"
-0.0 "--sync--" sync /:Engage!/ window 0,1
+0 "Start" sync /Engage!/ window 0,1
+0.8 "--sync--" sync /:Cloud Of Darkness:5627:/
+16.9 "Ground-Razing Particle Beam" sync /:Cloud Of Darkness:5625:/
+27.1 "The Art Of Darkness" sync /:Cloud Of Darkness:5B4[56]:/
```

Comment:
I wonder a little bit if these should have a (L/R) or (P/S) for protean/stack on the single ones?